source/nodejs/adaptivecards-designer/README.md

```diff
@@ -207,7 +207,10 @@ ACDesigner.Strings.toolboxes.propertySheet.title = "Custom property sheet title"
 ACDesigner.Strings.toolboxes.sampleDataEditor.title = "Custom sample data editor title";
 ACDesigner.Strings.toolboxes.toolPalette.title = "Custom tool palette title";
 
-/* Modify the Element toolbox (BEFORE calling attachTo) */ 
+/* To configure custom Pic2Card endpoint */
+ACDesigner.Pic2Card.pic2cardService = "https://<<your-pic2Card-service-endpoint>> ";
+
```

Comment:
Per the other comment about adding a `privacyLink` property, can we also document that here in the README, since people who use this service are most likely going to want to configure this property at the same time